Responmessenger.css

```diff
@@ -0,0 +1,191 @@
+html,body{
+    margin: 0;
+    padding: 0;
+    font-family: 'Roboto', sans-serif;
+}
+.cover{
+    max-width: 1024px;
+    margin:0 auto;
+}
+
+.menu{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    align-items: center;
+    height: 80px;
+    position: sticky;
+    top: 0;
+    border-bottom: 1px solid lightgray;
+    background-color: #fafafa;
+    opacity: 0.9;
+}
+
+.logocontainer{
+    width: 40px;
+    height: 40px;
+}
+
+.logo{
+    width: 100%;
+}
+
+.meunItems{
+    text-decoration: none;
+    color:inherit;
+    margin-left: 40px;
+}
+
+.meunItems:hover{
+    border-bottom-style: solid;
+    border-bottom-color: #385898;
+    
+}
+
+.mobilecontainer{
```

Comment:
[See this comment](https://github.com/MMyatKhine/Messenger_Site_Copy/pull/1/files#r671484445)

Reply:
Sir, I don't know why I can't see this comment but I think it might be about to charge into  camelCasing style